README.md

```diff
@@ -25,10 +25,12 @@ pip install avotes-parser-cli
 2. From repository:
 
 ```shell
-git clone https://github.com/DmitIv/EVMScriptParser.git
-cd EVMScriptParser
+git clone git@github.com:lidofinance/AVotesParser.git
+cd AVotesParser
+
 cd production/avotes-parser-core
 python setup.py install
```

Comment:
Probably should change to poetry install or smth. There is no setup.py anymore

Reply:
I see, give me a few seconds :+1: